src/core/Akka/Actor/ActorCell.DefaultMessages.cs

```diff
@@ -207,7 +207,7 @@ private void ReceiveSelection(ActorSelectionMessage m)
         private int CalculateState()
         {
             if(IsWaitingForChildren) return SuspendedWaitForChildrenState;
-            if(Mailbox.IsSuspended()) return SuspendedState;
+            if(Mailbox!.IsSuspended()) return SuspendedState;
```

Comment:
`Mailbox` cannot be `null` at this point - otherwise we have had a catastrophic actor initialization failure and this actor will be killed before ever reaching this point.